hw/ip/flash_ctrl/data/flash_ctrl.sv.tpl

```diff
@@ -1179,21 +1189,26 @@ module flash_ctrl
   logic [flash_ctrl_pkg::BusAddrW-1:0] flash_host_addr;
 
   import prim_mubi_pkg::mubi4_test_true_loose;
-  logic host_disable;
-  logic disabled_rvalid;
-  logic [1:0] disabled_err;
+  lc_ctrl_pkg::lc_tx_t host_enable;
+
 
   // if flash disable is activated, error back from the adapter interface immediately
-  assign host_disable = mubi4_test_true_loose(flash_disable[HostDisableIdx]);
-  assign disabled_err = {2{disabled_rvalid}};
+  assign host_enable = mubi4_test_true_loose(flash_disable[HostDisableIdx]) ?
+                       lc_ctrl_pkg::Off :
+                       lc_ctrl_pkg::On;
```

Comment:
This is probably a good candidate for the new remapping function as well...

Reply:
great catch. I completely forgot about this.

Reply:
this will be done as part of #11692